src/main/java/dev/j3fftw/litexpansion/utils/Constants.java

```diff
@@ -10,6 +10,12 @@
 
     public static final int CUSTOM_TICKER_DELAY = SlimefunPlugin.getCfg().getInt("URID.custom-ticker-delay");
 
+    public static final double SUGAR_CANE_WATERING_SUCCESS_CHANCE = LiteXpansion.getCfg().getDouble("options.sugar-cane-watering-success-chance");
+
+    public static final double CROP_WATERING_SUCCESS_CHANCE = LiteXpansion.getCfg().getDouble("options.crop-watering-success-chance");
+
+    public static final double TREE_WATERING_SUCCESS_CHANCE = LiteXpansion.getCfg().getDouble("options.tree-watering-success-chance");
```

Comment:
I would highly suggest to avoid using the LX config here, it may be better to use an `ItemSetting<Double>` for your WateringCan here since these settings are strictly tied to an item and not general LX settings.

Reply:
Would a separate category within the LX config be fine too? i.e. `watering-can.tree-watering-success-chance`

Reply:
No, there is a very good reason why `ItemSetting` exists, you should use it for these cases.